DC/Lab10/notebook/DCLayers.py

```diff
@@ -0,0 +1,498 @@
+#import sys
```

Comment:
can you please move this to the `DC/` folder? (we don't need to have more layers, keeping the tree to 1 level is ideal.)